pkg/update/check.go

```diff
@@ -46,15 +48,16 @@ type source interface {
 	get() (*remoteResponse, error)
 }
 
-// EnvDisableUpdateCheck is environmental variable to disable remote
+// EnvDisableUpdateCheck is an environmental variable to disable remote
 // http request to check if a newer version of the CLI is available
 const EnvDisableUpdateCheck = "HPECLI_DISABLE_UPDATE_CHECK"
 
 // json file that describes the latest release version.  Should be updated when new versions are published
 // can alternatively change to using github tags once we real releases
 const versionHost = "raw.githubusercontent.com"
 
-const versionPath = "/HewlettPackard/hpecli/master/site/published-version.json"
+const versionPath = "/HewlettPackard/hpecli/Didier/MultiOSSupport/site/published-version.json"
```

Comment:
Since this is being merged to master... this needs to go back to the master branch